C/jets.c

```diff
@@ -405,6 +405,62 @@ RIGHTMOST_(64,8)
 RIGHTMOST_(64,16)
 RIGHTMOST_(64,32)
 
+#define LEFT_PAD_LOW_1_(bitsM)                                                 \
+/* left_pad_low_1_m : TWO |- TWO^m */                                          \
+bool left_pad_low_1_##bitsM(frameItem* dst, frameItem src, const txEnv* env) { \
+  (void) env; /* env is unused. */                                             \
+  bool bit = readBit(&src);                                                    \
+  write##bitsM(dst, bit);                                                      \
+  return true;                                                                 \
+}
+LEFT_PAD_LOW_1_(8)
+LEFT_PAD_LOW_1_(16)
+LEFT_PAD_LOW_1_(32)
+LEFT_PAD_LOW_1_(64)
+
+#define LEFT_PAD_LOW_(bitsN, bitsM)                                                    \
+/* left_pad_low_n_m : TWO^n |- TWO^m */                                                \
+bool left_pad_low_##bitsN##_##bitsM(frameItem* dst, frameItem src, const txEnv* env) { \
+  (void) env; /* env is unused. */                                                     \
+  for(int i = 0; i < (bitsM)/(bitsN) - 1; ++i) { write##bitsN(dst, 0); }               \
```

Comment:
a9f2c88d13783ee145c73d1545ee7a201edf01c9: `(bitsM)/(bitsN)`

This is a potential divide by zero. We have to use the macro correctly to exclude this case. Is this a problem? Should we add a warning in the docs?